lib/jsdom/browser/Window.js

```diff
@@ -377,7 +417,7 @@ function Window(options) {
       delete this._document;
     }
 
-    stopAllTimers(currentWindow);
+    this.__stopAllTimers();
```

Comment:
Is monkey-patching a concern?

Reply:
Not really. We've basically always "reserved" underscore props, it's nowadays taken mostly care of by webidl2js, but for a long time, that's just how it always was. I hope one day it'll be window's turn.